<launch>
  <arg name="stereo" default="stereo_down/scaled" />
  <arg name="image" default="image_rect" />

  <node name="object_detector" pkg="object_detection" 
        type="features2d3d_detector_node" output="screen" >
    <remap from="stereo" to="$(arg stereo)" />
    <remap from="image" to="$(arg image)" />
    <param name="normalize_illumination" value="true" />
    <param name="equalize_histogram" value="true" />
    <param name="matching_threshold" value="0.95" />
    <param name="stereo_matching_threshold" value="0.95" />
  </node>

</launch>
